bismuthcore/Utils/convert_db.py

```diff
@@ -87,11 +87,11 @@ def create(db, sql: tuple):
 
     try:
         remove('ledger_new.db')
-    except:
+    except Exception:
```

Comment:
I do agree a bare "except" is not PEP, but be aware "except Exception" does not catch the same things as except alone. "except BaseException" would catch the same events.
Deciding which except should catch a KeyboardInterrupt or not may be non trivial.
In doubt, I would not change the current working, therefore use BaseException if really the bare except is too hard for some eyes.
A proper exception handling should anyway detail the possible exceptions more in depth, and would bloat the codebase quite a bit so I would not advise it.

Reply:
Perhaps throwing in BaseException for catching system-exiting exceptions, above the `except Exception` should address this concern without bloating the codebase too much.